### PR TITLE
Remove all the variants

### PR DESCRIPTION
**Pull request by vanhauser-thc**

### CODE REVIEW

**Comment by vanhauser-thc** [comment]
Also untouched fuzzer fail with same benchmark openthread-2019-12-23:
```
#19 41.19 thread/link_quality.cpp:225:23: error: implicit conversion loses integer precision: 'int' to 'int8_t' (aka 'signed char') [-Werror,-Wimplicit-int-conversion]
#19 41.19     return linkmargin + aNoiseFloor;
#19 41.19     ~~~~~~ ~~~~~~~~~~~^~~~~~~~~~~~~
#19 41.19 1 error generated.
```
how e.g. afl can fail but aflfast pass is beyond me :)

I think this can be merged.

**Comment by inferno-chromium** [comment]
Can you please rebase .github/workflows/fuzzers.yml
 from latest master (some new entropic experiments are in, need a slight rebase.

**Comment by vanhauser-thc** [comment]
@inferno-chromium done!

File: fuzzers/aflplusplus/fuzzer.py [hunk 1]
```diff
@@ -173,8 +167,8 @@ def fuzz(input_corpus, output_corpus, target_binary, flags=tuple()):
     # os.environ['AFL_PRELOAD'] = '/afl/libdislocator.so'
 
     flags = list(flags)
-    if not flags or not flags[0] == '-Q' and '-p' not in flags:
-        flags += ['-p', 'seek']
+    #if not flags or not flags[0] == '-Q' and '-p' not in flags:
```
Comment (inferno-chromium) [inline fuzzers/aflplusplus/fuzzer.py:170]: Just to confirm that this is intentional.

**Review by inferno-chromium — APPROVED** [review]
Will merge once CI succeeds

File: fuzzers/aflplusplus/fuzzer.py (hunk 1 above)
Comment (vanhauser-thc) [reply fuzzers/aflplusplus/fuzzer.py:170]: seek is the new default so this is not needed atm.
however as a bug affected all schedules I have to rerun a schedule test and that might result in a new default :)

**Comment by vanhauser-thc** [comment]
@inferno-chromium the same failures again as above ... openthread is failing for a few fuzzers, however has nothing to do with this PR

**Comment by inferno-chromium** [comment]
> @inferno-chromium the same failures again as above ... openthread is failing for a few fuzzers, however has nothing to do with this PR

That is fine, we will debug later. Merging for now. Also, can you check if you are committing from right account, i dont know why cla is not showing as signed from this email.

**Comment by vanhauser-thc** [comment]
@inferno-chromium 
that 2nd account is "aflpp" a generic afl++ dev account I set up to be able to commit from our fuzzing server that is shared with the other from our team. Although I signed the CLA also with this account it is not registering :(

**Pull request merged**